`make proto` does not run

```
$ make dependencies
# ...
$ make proto
./tools/bin/buf generate
Failure: plugin grpc-gateway-ts: could not find protoc plugin for name grpc-gateway-ts
make: *** [Makefile:189: proto] Error 1
```

This was encountered before (https://weaveworks.slack.com/archives/C03EM05MZU3/p1655289843122999) but evidently never fixed. I suspect everyone just ran `go install ...` to get past the breakage, and never thought of it again.

--- a/Makefile
+++ b/Makefile
@@ -131,6 +131,7 @@ godeps=$(shell go list -deps -f '{{if not .Standard}}{{$$dep := .}}{{range .GoFi
 
 dependencies: ## Install build dependencies
 	$(CURRENT_DIR)/tools/download-deps.sh $(CURRENT_DIR)/tools/dependencies.toml
+	@go install github.com/grpc-ecosystem/protoc-gen-grpc-gateway-ts
 
 lint:
 	bin/go-lint

